mix.exs

```diff
@@ -38,7 +39,7 @@ defmodule Expo.MixProject do
   defp package do
     %{
       licenses: ["Apache-2.0"],
-      maintainers: ["Jonatan Männchen"],
+      maintainers: ["Jonatan Männchen", "José Valim", "Andrea Leopardi"],
```

Comment:
Given that much of this code comes from the original Gettext implementation that José and I worked on, I think this makes sense?

Reply:
@whatyouhide Agreed.